shot-consumer-compose/app/build.gradle

```diff
@@ -59,7 +59,7 @@ android {
 }
 
 shot {
-    tolerance = 0.6 //0.6%
+    tolerance = 1.1 //1.1%
```

Comment:
Just out of curiosity: why did the tolerance change from 0.6 to 1.1?

Reply:
For our compose, tests the rendering method used depends on the OS version we use. The difference between API 28 and API 30 in this scenario is basically the number of shadow pixels and antialiasing pixes changing whenever we run our tests on the CI environment or our local setup. There are just a few pixels modified, but we have a small screenshot forcing us to increase the tolerance value.